apps/meteor/ee/app/livechat-enterprise/server/business-hour/Multiple.ts

```diff
@@ -154,9 +154,10 @@ export class MultipleBusinessHoursBehavior extends AbstractBusinessHourBehavior
 		const department = await LivechatDepartment.findOneById<Pick<ILivechatDepartment, 'businessHourId'>>(departmentId, {
 			projection: { businessHourId: 1 },
 		});
-		if (!department || !agentsId.length) {
+		if (!department?.businessHourId || !agentsId.length) {
```

Comment:
should this be handled by `handleRemoveAgentsFromDepartments`? it looks like it partially does already, but more importantly it seems it does other things that doesn't depend on `businessHourId`